### DISCUSSION TITLE
installation stuck in Terminal

### DISCUSSION THREAD

**Discussion by aloa71** [simulot/immich-go#490] [Q&A]
Hello folks,

Maybe it will be a simple issue but I'm quite new in command line execution. I'm running on MacOS sequoia on Mac M1 and I've downloaded the immich-go_Linux_arm64 version. First when I tried to run the exec file, it says "zsh: exec format error:", I've tried with all linux releases available but same. I've authorized the command to run in security and privacy as mentioned in installation section, same.

Second point, it is probably linked to the first issue, when I try to run the command line tu upload a folder on my immesh server, it says access denied
here the command run ~/Desktop ./immichgo -server=http://192.168.1.19:YYYY -key=APIKEY upload ~/Downloads/export/“FOLDER”/ and it returns access denied to desktop

**Comment by simulot** [comment]
You have to download the macOS version of the executable file suitable for a M1, likely this one: https://github.com/simulot/immich-go/releases/download/0.22.0/immich-go_Darwin_arm64.tar.gz

Be sure you have authorized the execution of this file

**Reply by aloa71** [reply]
Ok my mistake, Darwin seems to be the good file but I've got another issue when running the exec. I still have the access denied issue when running the command. I've all authorized in security management for the Immich-go exec file... 
![Capture d’écran 2024-09-28 à 19 52 19](https://github.com/user-attachments/assets/10100fd0-bd76-49bd-926b-2240544e059b)
![Capture d’écran 2024-09-28 à 19 52 42](https://github.com/user-attachments/assets/49ce7656-cab0-4e3f-a426-b01ff121765f)

**Comment by simulot** [comment]
The 1st capture show that immich-go is launched without parameter, hence the message "missing command"

On the 2nd capture, I think that the executable file isn't not allowed to run. ➡️ https://github.com/simulot/immich-go?tab=readme-ov-file#executing-immich-go
